src/style.css

```diff
@@ -1,3 +1,70 @@
-body {
+body,
+ul,
+li {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --header-heigth: 60px;
```

Comment:
There is a typo in the CSS variable `--header-heigth`. It should be `--header-height` to ensure consistency and avoid confusion.